src/java.base/share/classes/jdk/internal/foreign/abi/fallback/LibFallback.java

```diff
@@ -233,6 +243,11 @@ private NativeConstants() {}
         static final MemorySegment FLOAT_TYPE = MemorySegment.ofAddress(ffi_type_float());
         static final MemorySegment DOUBLE_TYPE = MemorySegment.ofAddress(ffi_type_double());
         static final MemorySegment POINTER_TYPE = MemorySegment.ofAddress(ffi_type_pointer());
+        static final int SIZEOF_SHORT = ffi_sizeof_short();
```

Comment:
Decided to change this a bit - e.g. instead of returning FFI type (which we don't use), I return sizeof instead, which seems more directly usable. This leads to some code simplifications (see definition of SIZE_T in FFIType).